plugins/service-monitor-cloudera/service-monitor-cloudera-service/src/main/java/com/thinkbiganalytics/servicemonitor/rest/client/RestClientConfig.java

```diff
@@ -29,6 +29,7 @@
     private String password = "PASSWORD";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'password' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)